source/develop/dev-process/working-with-gerrit.md

```diff
@@ -256,12 +252,34 @@ A [topic branch](http://progit.org/book/ch3-4.html) is a short-lived branch that
         url = http://gerrit.ovirt.org/p/project
         pushurl = ssh://username@gerrit.ovirt.org:29418/project.git
         fetch = +refs/heads/*:refs/remotes/gerrit/* 
+##### Installation
+Linux:
+
+For distributions like Debian, Ubuntu, or Mint open a terminal and type:
+
+`sudo apt install git-review`
+
+For distributions like RedHat, Fedora 21 or earlier, or CentOS open a terminal and type:
+
+`sudo yum install git-review`
+
+For Fedora 22 or later open a terminal and type:
+
+`sudo dnf install git-review`
+
+For SUSE distributions open a terminal and type:
+
+`sudo zypper in python-git-review`
```

Comment:
As above, depends on the openstack removal.

Reply:
I think we don't need to be so specific on how to intall git review, I assume a develper knows how to install a package on their own system :-)

Reply:
@sandrobonazzola  I agree 
I changed this:
removed specific installation and added link to OpenStack tutorial for installation

(But also can be totally removed)